src/Shells/StrixMusic.Shells.ZuneDesktop/Controls/Views/Collection/ZuneTrackCollectionItem.cs

```diff
@@ -132,32 +116,5 @@ private void TrackCollection_TracksChanged(object sender, IReadOnlyList<OwlCore.
                 ShouldShowArtistList = _lastKnownTrackArtistsCount.Any(x => x.Value > 1) && _parentCollection is IAlbum or IArtist;
```

Comment:
There's a lot going on in this file now, that doesn't need to. 

`ShouldShowArtistList` can be a normal property with a getter / setter, and can be set from the consuming ZuneTrackCollection instead of here (less overhead that way).